ext/xml/tests/xml_parser_get_option_variation4.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+xml_parser_get_option() - Test that a parameter not set defaults
+and returns a E_WARNING-level message with "Unknown option" text
+--CREDITS--
```

Comment:
Please, do no longer use the CREDITS section; see https://github.com/php/php-src/blob/master/CONTRIBUTING.md#writing-tests.

Reply:
Done. I hadn't read that section in a few years.